app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -168,11 +174,9 @@ private void showFeed() {
         txtvTitle.setText(feed.getTitle());
 
         String description = feed.getDescription();
-        if(description != null) {
-            if(Feed.TYPE_ATOM1.equals(feed.getType())) {
-                HtmlToPlainText formatter = new HtmlToPlainText();
-                Document feedDescription = Jsoup.parse(feed.getDescription());
-                description = StringUtils.trim(formatter.getPlainText(feedDescription));
+        if (description != null) {
+            if (Feed.TYPE_ATOM1.equals(feed.getType())) {
+                description = getPlainText(new SpannedString(description)).toString();
```

Comment:
getPlainText needs to be called even if the feed is not atom1. Otherwise, your PR does not fix the original issue (that's not an atom1 feed)